resources/lang/fr/Ticket.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+return array (
+  //============================== New strings to translate ==============================//
+  // Defined in file C:\\wamp\\www\\attendize\\resources\\views\\ManageEvent\\Tickets.blade.php
+  'on_sale' => 'On Sale',
+  //==================================== Translations ====================================//
+  'attendee_ref' => 'Réf. participant',
+  'coupon_codes' => 'Codes Coupon',
+  'create_ticket' => 'Créer un billet',
+  'demo_attendee_ref' => '#YLY9U73-1',
+  'demo_end_date_time' => 'Mar 18th 5:08PM',
```

Comment:
?

Reply:
Please, can you commit fixes directly?

Reply:
Or be a bit less terse about the errors at stake? For this error above, I don't understand what you don't understand. Thanks.

Reply:
The datetime is still in english. Maybe it will be better like this : "18 mars 17:08" but I'm not sure about the format used when rendering a ticket

Reply:
@jeremyquinton Shall I localize `'Mar 18th 5:08PM'`?